SRVKS-951: Add internal TLS enablement docs (#50404)
openshift/openshift-docs: serverless/security/serverless-config-tls.adoc

:_content-type: ASSEMBLY
[id="serverless-config-tls"]
= Configuring TLS authentication
include::_attributes/common-attributes.adoc[]
:context: serverless-config-tls

toc::[]

> You can use _Transport Layer Security_ (TLS) to encrypt Knative traffic, as well as for authentication.

Reviewer: ```suggestion
You can use _Transport Layer Security_ (TLS) to encrypt Knative traffic and for authentication.
```
[ISG nit](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#as-well-as)
